nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/processors/excel/SplitExcel.java

```diff
@@ -230,6 +229,32 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
         session.transfer(flowFileSplits, REL_SPLIT);
     }
 
-    private record WorkbookSplit(int index, FlowFile content, String sheetName, int numRows) {
+    private int copyRows(final Sheet originalSheet, final SXSSFSheet destinationSheet) {
+        final CellCopyContext cellCopyContext = new CellCopyContext();
+        int rowCount = 0;
+
+        for (final Row sourceRow : originalSheet) {
+            final Row destinationRow = destinationSheet.createRow(sourceRow.getRowNum());
+            destinationRow.setHeight(sourceRow.getHeight());
+
+            for (final Cell sourceCell : sourceRow) {
+                final Cell destCell = destinationRow.createCell(sourceCell.getColumnIndex());
+                CellUtil.copyCell(sourceCell, destCell, CELL_COPY_POLICY, cellCopyContext);
+            }
+
+            ++rowCount;
```

Comment:
Is there a reason for the prefixed `++` as opposed to the suffix approach?
```suggestion
            rowCount++;
```

Reply:
I don't think so, changed that one to suffix